derp/derp_server.go

```diff
@@ -318,6 +323,7 @@ func NewServer(privateKey key.NodePrivate, logf logger.Logf) *Server {
 		avgQueueDuration:     new(uint64),
 		tcpRtt:               metrics.LabelMap{Label: "le"},
 		keyOfAddr:            map[netip.AddrPort]key.NodePublic{},
+		clock:                clock,
```

Comment:
this is the only use of the global? I'd get rid of the global and set this to StdClock explicitly here and let tests change it as needed.